### PR TITLE
add tableMeta to customRender function

### PR DESCRIPTION
**Pull request by gregnb**
When merged this should address #63 

@jkantr and @samsch would love some feedback on this PR.

### CODE REVIEW

File: src/MUIDataTable.js [hunk 1]
```diff
@@ -269,47 +270,62 @@ class MUIDataTable extends React.Component {
...
+        );
+        columnDisplay = funcResult;
+
+        /* drill down to get the value of a cell */
+        columnValue =
           typeof funcResult === "string"
             ? funcResult
             : funcResult.props && funcResult.props.value
               ? funcResult.props.value
               : column;
```
Comment (samsch) [inline src/MUIDataTable.js]: There's an orphaned reference to `column` here. I think that should be `columnValue`?

Maybe add `no-undef` rule to eslint config?

Comment (gregnb) [reply src/MUIDataTable.js]: Nice catch. I've updated that and also added some additional eslint rules

**Comment by gregnb** [comment]
TODO: Update README to reflect changes

**Comment by samsch** [comment]
I had a little time to put together at least part of what's needed for the docs. Probably isn't formatted as you'll want, but might save you a bit of work: https://github.com/gregnb/mui-datatables/commit/e19fb3610e8504d5c487769e7eb3953ae80cb3d7

**Comment by gregnb** [comment]
Sorry for the delay have a lot going on right now. I will review this one more time tonight and then merge it in

**Pull request merged**